utilities/openemr-cmd/openemr-cmd

```diff
@@ -24,267 +30,292 @@ get_container_names() {
 # Set the container names
 get_container_names
 
+# Function to run the devtools script inside the specified container
+# Usage: run_devtools_in_docker <container_id> <devtools_command> [args...]
+run_devtools_in_docker() {
+  local container_id="$1"
+  shift # Remove container_id from arguments
+  # DEV_TOOLS is '/root/devtools'
+  # "$@" passes the remaining arguments (devtools command and its args) correctly quoted
+  docker exec -i "${container_id}" "${DEV_TOOLS}" "$@"
+}
+
+# Function to run an arbitrary shell command string inside the specified container
+# Usage: run_shell_command_in_docker <container_id> <command_string>
+run_shell_command_in_docker() {
+  local container_id="$1"
+  local command_string="$2"
+  docker exec -i "${container_id}" sh -c "${command_string}"
+}
+
 # Set some constants
-DOCKER_EXEC_CMD='docker exec -i'                                 # docker exec command
-SHELL_TYPE='sh -c'                                               # specify shell type
-FIRST_ARG=$1											                               # define the first parameter
-DEV_TOOLS=/root/devtools
+FIRST_ARG="${1:-}" # define the first parameter, default to empty if not set
+DEV_TOOLS='/root/devtools'
 
 check_docker_compose_install_or_not(){
-	DOCKER_COMPOSE_CODE=22
-	which docker-compose &>/dev/null
-	[ $? -ne 0 ] && echo "Please check docker-compose install or not." && exit $DOCKER_COMPOSE_CODE
+	local DOCKER_COMPOSE_CODE=22
```

Comment:
I'll make a PR to bring the .editorconfig from openemr/openemr over so we can start to try to avoid mixing tabs and spaces, and other stuff.

Reply:
Just made a change that should address that specific area. Lots of mixing of tabs & spaces in here.